pkg/driver/camera/camera_linux.go

```diff
@@ -92,21 +101,21 @@ func (c *camera) VideoRecord(p prop.Media) (video.Reader, error) {
 
 	r := video.ReaderFunc(func() (img image.Image, err error) {
 		// Wait until a frame is ready
-		for {
+		for i := 0; i < maxErrorCount; i++ {
 			err := c.cam.WaitForFrame(5)
 			switch err.(type) {
 			case nil:
 			case *webcam.Timeout:
-				continue
+				return nil, errReadTimeout
```

Comment:
Is this supposed to be `continue` maybe?

Reply:
Umm, in my feeling, 5 seconds of the timeout is long enough to treat as an error.

Reply:
Oops. My bad. That makes sense. Yes, 5 seconds is way too long... It should be an error.